cmake/schemes/url_sha1_boost_ios_library.cmake.in

```diff
@@ -148,6 +149,16 @@ set(
     --user-config="${boost_user_jam}"
 )
 
+hunter_boost_component_b2_args(
+  "@HUNTER_PACKAGE_COMPONENT@"
+  "@HUNTER_Boost_CMAKE_ARGS@"
+  b2_component_opts
+  )
+
+if(b2_component_opts)
```

Comment:
actually do we need this one here, if we add empty option I guess nothing bad happens (?)